app/core/vueRouter.js

```diff
@@ -31,6 +32,18 @@ export default function getVueRouter () {
           component: PageInteractiveEditor,
           props: true
         },
+        {
+          path: '/ozaria/play/intro/:introLevelIdOrSlug?',
```

Comment:
<33333333333333

Reply:
Do we need to remove the page layout for this?

Reply:
Yeah, it is currently rendered with header and footer, will update accordingly once we have support for empty page layouts.

Reply:
Awesome - I think its a great decision to put this in here before having the page layout setup so I'm two thumbs up.